source/_integrations/zha.markdown

```diff
@@ -214,6 +214,7 @@ Some devices can be auto-discovered, which can simplify the ZHA setup process. T
 | [ZigStar Coordinators](https://zig-star.com/)                                                                                               | USB              | 1A86:7523                      |
 | [SMLIGHT SLZB-06 POE Zigbee LAN WiFi USB Adapter](https://smlight.tech/product/slzb-06/)                                                    | Zeroconf         | slzb-06.local.                 |
 | [ZigStar UZG Universal Zigbee Gateway (UZG-01)](https://uzg.zig-star.com)                                                                   | Zeroconf         | uzg-01._tcp.local.             |
+| [cod.m Zigbee Coordinator](https://docs.codm.de/zigbee/zigbee-coordinator/)                                                                 | Zeroconf         | czc._tcp.local.                |
```

Comment:
This URL redirects to https://docs.codm.de/zigbee/coordinator/ now.
Should this be updated or is this intentional and should stay as-is?

Reply:
Thanks for the hint, changed